huawei-cloud/servicestage/src/main/java/org/apache/servicecomb/huaweicloud/servicestage/CasEnvConfig.java

```diff
@@ -57,10 +62,31 @@ private void init() {
         .getInstance().getStringProperty(INSTANCE_ID, EMPTY).get());
     properties.put(ENVIRONMENT_ID, DynamicPropertyFactory
         .getInstance().getStringProperty(ENVIRONMENT_ID, EMPTY).get());
+    properties.put(SERVICE_PROPS, DynamicPropertyFactory
+            .getInstance().getStringProperty(SERVICE_PROPS, EMPTY).get());
+    properties.put(INSTANCE_PROPS, DynamicPropertyFactory
+            .getInstance().getStringProperty(INSTANCE_PROPS, EMPTY).get());
```

Comment:
Maybe read this 2 properties in getNonEmptyInstanceProperties and getNonEmptyServiceProperties is better.

Reply:
OK